### PR TITLE
[WIP] Add protocol entry for server status

### PR DESCRIPTION
**Pull request by Daenyth**
## Related Ticket(s)
- Ref #2474

## What will change with this Pull Request?
WIP branch to get the protocol + servatrice implementation written.
Cockatrice implementation and UI will come in another ticket.

Don't merge yet

### CODE REVIEW

File: common/pb/response_server_status.proto [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    required int32 connected_users = 3;
+    optional int32 max_users = 4;
+
+    required string server_name = 5;
+    optional string server_description = 6;
+    optional string server_admin_contact = 7;
+    optional string server_website = 8;
+
+    repeated string required_features = 9;
+    repeated int32 cockatrice_ports = 10;
```
Comment (tooomm) [inline common/pb/response_server_status.proto]: `servatrice_ports`?

Comment (ctrlaltca) [reply common/pb/response_server_status.proto]: i guess the name choice is something like "ports that cockatrice can use to connect", while the next field is "ports that webatrice can use to connect"

Comment (Daenyth) [reply common/pb/response_server_status.proto]: @ctrlaltca has it exactly.

Comment (Daenyth) [reply common/pb/response_server_status.proto]: I'll add comments to everything

**Comment by tooomm** [comment]
https://www.reddit.com/r/Cockatrice/comments/92pm9z/cant_connect_to_cockatricees/

>i cant connect to (xyz).
>
>Is the server down at the moment? Or is the problem on my end?

<br>

I'm so looking forward to this...

**Comment by ctrlaltca** [comment]
@Daenyth, this looks ok, are you fine with it?

**Comment by Daenyth** [comment]
I am but I'd like to not merge and work on the servatrice implementation on
this branch just in case we decide to change the proto. I don't want an
interim version going out in a beta or something

**Comment by ZeldaZach** [comment]
@ctrlaltca Do we want to do anything with this?

**Comment by ctrlaltca** [comment]
Yep, sooner or later I'll take on and develop it.

**Comment by ZeldaZach** [comment]
Closing for the time being. @ctrlaltca if you find some time, feel free to give it a shot. Will leave the branch up.

**Pull request closed without merging**